.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "typedoc-template"]
```

Comment:
I had to think about this for a bit! So that I understand how this works, when we do `npm pack` on service tooling, the openfin typedoc templete will get included in the package?

Reply:
Submodules are a bit weird overall, but what I have done is put this dir in the `src` directory so it will be included in the pack.  This `.gitmodules` exists only so we can keep it up to date